package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "workspaces": ["src", "example"],
+  "version": "1.0.0",
```

Comment:
I think it's needed by yarn to install this as a package https://github.com/yarnpkg/yarn/issues/6195